### PR TITLE
Issue 60 tailwind credit

### LINKED ISSUE #59: Add a thank you to 1Password for sponsoring a Team account for us
> All setup! I've applied the promotion to your account. Just as a heads up, we review the promotion once a year. If you could reach out in just under 12 months, we can see about extending things before your account becomes frozen.
> 
> As far as our branding, there are some assets here which should help: https://1password.com/press/#press-kit. No particular preferences there, feel free to use whatever you think looks best. Linking to our homepage sounds great.
> 
> Hope that all makes sense! Let me know if you have any more questions, and in the meantime have a brilliant weekend.

### LINKED ISSUE #60: Add a thank you to TailwindUI to our website
We need to track down branding details, but a logo and link to their website seems like a good place to start

### PR DESCRIPTION
**Pull request by dboydgit**
Adds a sponsors page with links to Tailwind CSS and 1Password. Closes #59 and closes #60

### CODE REVIEW

File: components/ButtonLink.vue [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default {
     props: {
         href: {
             type: String,
-            default: "#0",
+            default: "#",
```
Comment (MarkBennett) [inline components/ButtonLink.vue:26]: I'm not familiar enough with Vue to understand the purpose of this change. Is this just the default empty hash? Was this just a typo before?

File: components/TheFooter.vue [hunk 2]
```diff
@@ -1,90 +1,142 @@
 <template>
-    <footer class="bg-brand-grey">
-        <div class="container flex justify-around mx-auto p-4">
-            <a
-                class="link white-60 bg-transparent hover-white inline-flex items-center ma2 tc br2 pa2"
-                href="https://facebook.com/devedmontonsociety"
-                title="Dev Edmonton Society Facebook"
+    <!-- This example requires Tailwind CSS v2.0+ -->
```
Comment (MarkBennett) [inline components/TheFooter.vue:2]: Is this comment meant to be removed @dboydgit?

File: components/TheFooter.vue [hunk 3]
```diff
@@ -1,90 +1,142 @@
...
+    <footer class="bg-grey-000">
```
Comment (MarkBennett) [inline components/TheFooter.vue:3]: I really like the addition of some of the navigation to the footer. Nice work here @dboydgit

**Review by MarkBennett — CHANGES_REQUESTED** [review]
Looks great @dboydgit! Thanks for taking this on!

Just a few small nitpicks.

I was also think we should look at getting all the navigation links and URLs in one place in the codebase so they're easy to change in the future, but I don't think that's in scope for this PR. I'll open a seperate issue.

File: components/ButtonLink.vue (hunk 1 above)
Comment (dboydgit) [reply components/ButtonLink.vue:26]: Oh I thought this was a typo, but after a bit research I see that the `0` does actually provide some functionality. I'll revert this. Thanks!

File: components/TheFooter.vue (hunk 2 above)
Comment (dboydgit) [reply components/TheFooter.vue:2]: Yes we can take these out ✅

**Review by MarkBennett — APPROVED** [review]
Looks good @dboydgit! Ship it!

**Pull request merged**